docs/UserGuide.md

```diff
@@ -160,11 +161,36 @@ ModBook data is saved as a JSON file `[JAR file location]/data/modbook.json`. Ad
 Caution: If your changes to the data file makes its format invalid, ModBook will discard all data and start with an empty data file at the next run.
 
 ## FAQ
-**Q**:   How do I transfer my data to another Computer?
+**Q**:   How do I transfer my data to another Computer?  
 **A**:   Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous ModBook home folder.
 
+## Date and time formats
+The following formats are valid for entering time values:
+
+| Format   | Examples             |
+|----------|----------------------|
+| `HH:mm`  | `09:00`, `14:30`     |
+| `HH.mm`  | `09.00`, `14.30`     |
+| `HHmm`   | `0900`, `1430`       |
+| `ha`     | `9AM`, `2pm`         |
+| `hha`    | `09AM`, `11pm`       |
+| `h:mma`  | `9:00AM`, `2:30pm`   |
+| `hh:mma` | `09:00AM`, `11:30pm` |
+| `h.mma`  | `9.00AM`, `2.30pm`   |
+| `hh.mma` | `09.00AM`, `11.30pm` |
+
+The meaning of the symbols in the formats specified are in the table below:
+
+| Symbol | Meaning                                 |
+|--------|-----------------------------------------|
+| H      | Hour of 24-hour time. Goes from 0 to 23 |
+| h      | Hour of 12-hour time. Goes from 1 to 12 |
+| m      | Minute                                  |
+| a      | AM/PM indicator                         |
+
 ## Command Summary
 ### Command Summary by Action
+
```

Comment:
trying out to see whether adding a new line will make the table render properly in Jekyll

Reply:
uhh i think have to merge first?? dunno man

Reply:
sure we can try